src/SignalR/clients/java/signalr/src/main/java/com/microsoft/signalr/DefaultHttpClient.java

```diff
@@ -36,7 +36,7 @@ public void close() {
         }
     }
 
-    public DefaultHttpClient(int timeoutInMilliseconds, OkHttpClient client) {
+    public DefaultHttpClient(int timeoutInMilliseconds, OkHttpClient client, Action1<OkHttpClient.Builder> configureBuilder) {
```

Comment:
Can we remove the `timeoutInMilliseconds` parameter then? I don't see anywhere it's passed in with a null client, and with a non-null client it doesn't do anything.